README.md

```diff
@@ -115,6 +151,12 @@ When it launches it will send PR to update all the repos selected in step (2.2).
 
 </details>
 
+## Configuration
+
+There is a `repo-config` option in usage which is a path to a configuration file. It defaults to `.github/.scala-steward.conf`.
+
+See the [configuration documentation](https://github.com/scala-steward-org/scala-steward/blob/main/docs/repo-specific-configuration.md).
+
```

Comment:
I believe this is redundant, as it is already specified in the `Usage` section.

```suggestion
```

Reply:
I don't mind leaving it only in the usage section, but I'm going to add the link in there then, as it is in a separate repo.